tests/src/action.spec.ts

```diff
@@ -124,6 +126,21 @@ describe('src/action', () => {
 		expect(createTagStub.lastCall.args[1]).to.equal('v0.5.6');
 	});
 
+	it('passes correct build parameters to balena-utils', async () => {
+		// @ts-expect-error
+		await action.run(context, inputs);
+		// Check that the right parameters were passed
+		expect(pushStub.lastCall.firstArg).to.equal('my-org/my-fleet');
+		expect(pushStub.lastCall.args[1]).to.equal('undefined//src'); // TODO fix this string return
+		expect(pushStub.lastCall.lastArg).to.deep.equal({
+			draft: false,
+			noCache: false,
+			tags: {
+				sha: 'fba0317620597271695087c168c50d8c94975a29',
+			},
+		});
+	});
```

Comment:
I had to check the params individually because the last param was an array and it needs `deep.equal` or it kept failing for some unknown reason it wouldn't tell me...